glide.yaml

```diff
@@ -1,19 +1,9 @@
 package: github.com/giantswarm/aws-operator
 import:
 - package: github.com/aws/aws-sdk-go
-  version: master
-  subpackages:
-  - service/autoscaling
-  - service/s3
-  - service/elb
-  - service/route53
-  - awserr
+  version: v1.12.3
```

Comment:
There was a problem with the route53 package introduced in some master version. This caused the operator to fail when reconciling guest clusters. I pinned the latest version here. We should update at some point but for the external dependency some more control is a good thing.